common/client_test.go

```diff
@@ -16,7 +16,7 @@ func TestDatabricksClientConfigure_BasicAuth_NoHost(t *testing.T) {
 		Username: "foo",
 		Password: "bar",
 	}
-	err := dc.Configure()
+	err := dc.ConfigureWithAuthentication()
```

Comment:
assert for error effect by calling `.Get("/dummy", nil, nil)`, because this is where change actually is in production flow, not in `ConfigureWithAuthentication`, which is not called by provider itself and should be reverted.

Reply:
Alright, got it. This will work for this test, but for `TestDatabricksClientConfigure_BasicAuth` and `TestDatabricksClientConfigure_HostWithoutScheme` (basically all tests where we expect a success state) we'll need the mock server because the get request will fail otherwise. Is that okay?